app/src/main/java/cm/aptoide/pt/install/InstallManager.java

```diff
@@ -240,19 +241,25 @@ private boolean isAppInstalling(List<RoomInstalled> installingAppList, String pa
   }
 
   public Completable install(RoomDownload download) {
-    return install(download, false, false, true);
+    boolean shouldForceDefaultInstall =
+        sharedPreferences.getBoolean(ManagedKeys.ENFORCE_NATIVE_INSTALLER_KEY, false);
+    return install(download, shouldForceDefaultInstall, false, true);
   }
 
   public Completable install(RoomDownload download, boolean shouldInstall) {
-    return install(download, false, false, shouldInstall);
+    boolean shouldForceDefaultInstall =
+        sharedPreferences.getBoolean(ManagedKeys.ENFORCE_NATIVE_INSTALLER_KEY, false);
+    return install(download, shouldForceDefaultInstall, false, shouldInstall);
   }
 
   private Completable defaultInstall(RoomDownload download) {
     return install(download, true, false, true);
   }
 
   public Completable splitInstall(RoomDownload download) {
-    return install(download, false, true, true);
+    boolean shouldForceDefaultInstall =
+        sharedPreferences.getBoolean(ManagedKeys.ENFORCE_NATIVE_INSTALLER_KEY, false);
```

Comment:
same as above

Reply:
Also here, note for the future, but forceSplitInstall and forceDefaultInstall are mutually exclusive as we can only use one type of install for each install.